my-app/README.md

```diff
@@ -10,7 +10,7 @@ Finished certifications is checked:
 
 ## Reactjs.org
 
-Tic-tac-toe tutorial game is built and its source code [here](TODO:Add relative path link from the repository). Implementation goals:
+Tic-tac-toe tutorial game is built and its source code [/my-app/](TODO:Add relative path link from the repository). Implementation goals:
```

Comment:
why there's a duplicate readme? this one looks irrelevant. Top one is about the whole repository and internship. this one should be specific to the project. Initial `create-react-app`'s readme would be fine.